cartography/intel/gcp/storage.py

```diff
@@ -58,188 +62,98 @@ def get_gcp_buckets(storage: Resource, project_id: str) -> Dict:
 
 
 @timeit
-def transform_gcp_buckets(bucket_res: Dict) -> List[Dict]:
+def transform_gcp_buckets(bucket_res: Dict) -> Tuple[List[Dict], List[Dict]]:
```

Comment:
Given that we transform buckets and bucket labels here and load them separately, let's call this `transform_gcp_buckets_and_labels` to make things a bit clearer? Optionally we could separate this into two methods but I dont think that's required until this module uses generators